### DISCUSSION TITLE
Welcome to Elsa Guides Discussions!

### DISCUSSION THREAD

**Discussion by sfmskywalker** [elsa-workflows/elsa-guides#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Nao050101** [comment]


**Reply by sfmskywalker** [reply]
Hi @Nao050101, welcome!

Since this is the welcome page to discussions, it would be better if you ask your question by [creating a new discussion](https://github.com/elsa-workflows/elsa-guides/discussions/new).

Thank you!

**Reply by Nao050101** [reply]
Thank you.
